index.html

```diff
@@ -3604,6 +3604,37 @@ <h2>A project for first-time contributions</h2>
         </div>
         <!-- ________ Mariana Martins' card END ________  -->
 
+
+
+        <!-- ________ Michael card START ________  -->
+        <div class="card">
+          <p class="name">Olubimi michael olumide</p>
+          <p class="contact">
+            <i class="fab fa-github"></i>
+            <a href="https://github.com/olumide12-cell/" target="_blank">GitHub</a>
+            <i class="fab fa-linkedin"></i>
+            <a href="https://www.linkedin.com/in/olubimi-michael/" target="_blank">LinkedIn</a>
+          </p>
+          <p class="about">Hi, I'm Olumimi Michael Olumide, a full stack software engineer enthusiast, 
+            currently a software engineer student at ALX, a MERN stack web developer. I'm currently seeking a remote position; message me on LinkedIn and check out my repos to see what I've got to offer!</p>
```

Comment:
Can you review the about. I'm not sure if it's permitted to advertise yourself for a remote job. or is it permitted @Syknapse